changelog.d/7166.misc

```diff
@@ -0,0 +1 @@
+New App Layout is now enabled by default! Go to the Settings > Labs to toggle this
```

Comment:
Not a big deal, but the file name could be updated. Anyway let's not trigger the CI for such change. Please merge!